packages/primitives/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react'
+import { SxStyleProp } from 'theme-ui'
+import { VariantProps } from '@kodiak-ui/core'
+import { Box } from '../Box'
+import { Text } from '../Text'
+
+type Position = 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right'
+
+function getPositionStyles(position: Position): SxStyleProp {
+  switch (position) {
+    case 'top-right':
+      return {
+        top: 0,
+        right: 0,
+        transform: 'scale(1) translate(50%, -50%)',
```

Comment:
is `scale(1)` for the animation?

Reply:
I copied it from Material and I think it is used from going from a dot to a badge with a number. I've removed it fo rnow.